documentation/docs/core-concepts/core-packages.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Core Packages
+path: /core-packages/
+---
+
+# Core Packages
+
+Fusion.js is comprised of separate packages. Each package is responsible for a different part of the core framework and rather than ship as one monolithic package, the functionality is split to facilitate clear separation of concerns as well as easier upgrade paths for users.
+
+The `create-fusion-app` scaffold will set up a new Fusion.js app with these pre-installed. The [Learning Fusion.js Tutorial](/docs/learning-fusionjs-tutorial) walks through installing these manually and setting them up.
+
+Let's dive in and explore at a high level what these packages are.
+
+## fusion-core
+
+([GitHub](https://github.com/fusionjs/fusionjs/tree/master/fusion-core))
```

Comment:
These Github links get mangled in the preview deployment. Any idea if that will happen when it's deployed?

Reply:
Oh, probably. I'll have to fix the regex replacement. Good catch 👍